client/views/actions/invite.tpl

```diff
@@ -1,9 +1,9 @@
 {{> ../user_name nick=from}}
-invited
+{{translate 'client.invited.invited'}}
 {{#if invitedYou}}
-	you
+	{{translate 'client.invited.you'}}
 {{else}}
 	{{> ../user_name nick=invited}}
 {{/if}}
-to
+{{translate 'client.invited.to'}}
 {{{parse channel}}}
```

Comment:
Yeah, not too sure how to solve this -- need to do some reading up on Handlebars. http://i18next.github.io/i18next/pages/doc_templates.html has an "extended handlebars helper" but for the life of me I can't figure out how it works